packages/sanity/src/tasks/src/tasks/components/form/tasksFormBuilder/FormCreate.tsx

```diff
@@ -0,0 +1,95 @@
+import {Box, Flex, Switch, Text, useToast} from '@sanity/ui'
+import {useCallback, useState} from 'react'
+import {type ObjectInputProps, set} from 'sanity'
+
+import {Button} from '../../../../../../ui-components'
+import {useTasksNavigation} from '../../../context'
+import {useRemoveTask} from '../../../hooks/useRemoveTask'
+import {type TaskDocument} from '../../../types'
+import {getMentionedUsers} from '../utils'
+
+const getTaskSubscribers = (task: TaskDocument): string[] => {
+  const subscribers = task.subscribers || []
+
+  getMentionedUsers(task.description).forEach((user) => {
+    if (!subscribers.includes(user)) subscribers.push(user)
+  })
+
+  // Check if the task has been assigned, add the assignee to the subscribers list.
+  if (task.assignedTo) {
+    if (!subscribers.includes(task.assignedTo)) {
+      subscribers.push(task.assignedTo)
+    }
+  }
+  return subscribers
+}
+export function FormCreate(props: ObjectInputProps) {
+  const [createMore, setCreateMore] = useState(false)
+  const {
+    setViewMode,
+    setActiveTab,
+    state: {viewMode},
+  } = useTasksNavigation()
+
+  const toast = useToast()
+  const handleCreateMore = useCallback(() => setCreateMore((p) => !p), [])
+  const {onChange} = props
+  const value = props.value as TaskDocument
+  const onRemove = useCallback(() => {
+    setViewMode({type: 'list'})
+  }, [setViewMode])
+  const {handleRemove, removeStatus} = useRemoveTask({id: value._id, onRemoved: onRemove})
+
+  const handleCreate = useCallback(() => {
+    if (!value?.title) {
+      toast.push({
```

Comment:
This toast appears **behind** the task sidebar now, probably after Herman's changes that just got merged

Reply:
Seems a regression from https://github.com/sanity-io/sanity/pull/5949 
@hermanwikner could you please look at it?